src/frontend/staff-portal/src/app/components/jj-workbench/jj-dispute-digital-case-file/jj-dispute-digital-case-file.component.ts

```diff
@@ -56,16 +59,15 @@ export class JJDisputeDigitalCaseFileComponent implements OnInit, AfterViewInit
 
   applyFilter() {
     this.dataSource.filter = this.filterText;
+    this.tableHeight = this.calcTableHeight(300);
   }
 
   refreshData(jjDisputes: JJDispute[]): void {
     this.data = jjDisputes;
-    // only show status NEW, IN_PROGRESS, CONFIRMED, REVIEW, REQUIRE_COURT_HEARING, REQUIRE_MORE_INFO
-    this.data = this.data.filter(x => this.statusDisplay.indexOf(x.status) > -1 && x.hearingType === this.HearingType.CourtAppearance);
     this.dataSource.data = this.data;
```

Comment:
no need to set here as you set it in line 81

Reply:
will do